package.json

```diff
@@ -20,5 +20,5 @@
     "node": ">=18.17.0",
     "pnpm": ">=8.15.1"
   },
-  "packageManager": "^pnpm@8.15.1"
```

Comment:
I think we keep going back and forth on this caret, I was able to run `pnpm` commands with the caret on my end, are you able to?

Reply:
Looks like the caret is unsupported syntax, we can remove it
https://stackoverflow.com/questions/71747609/how-to-specify-packagemanager-in-package-json